app/uk/gov/hmrc/apiscope/controllers/package.scala

```diff
@@ -27,9 +27,8 @@ import uk.gov.hmrc.http.NotFoundException
 import uk.gov.hmrc.apiscope.models.ErrorCode.{API_INVALID_JSON, SCOPE_NOT_FOUND}
 import uk.gov.hmrc.apiscope.models.{ErrorCode, ErrorDescription, ErrorResponse}
 
-package object controllers {
+package object controllers extends ApplicationLogger {
```

Comment:
While this is clearly legal it looks (IMO) a bit weird. Up to you whether you want to change it